using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -680,16 +778,19 @@ If the current SDK privacy status is `optedout`, the advertising identifier is n
 
 {% tabs %}
 {% tab title="Android" %}
-#### Java
 
-### setAdvertisingIdentifier
+### setAdvertisingIdentifier<a id="setAdvertisingIdentifier-java"></a>
+
+This `setAdvertisingIdentifier()` API sets  the advertising identifier provided. 
 
 #### **Syntax**
 
 ```java
 public static void setAdvertisingIdentifier(final String advertisingIdentifier);
 ```
 
+o *advertisingIdentifier* is a String that provides developers with a simple, standard system to continue to track the Ads through their apps	 
```

Comment:
- Move parameter information to after the code example section.
- Reword so that it reads:
"* *advertisingIdentifier* is a string that provides developers with a simple, standard system to track ads through their apps."